marklogic-data-hub/src/test/java/com/marklogic/bootstrap/DataHubInstallTest.java

```diff
@@ -122,9 +123,27 @@ public void setup()
             } catch (IOException e) {
                 throw new RuntimeException(e);
             }
+            copyFlowArtifactsToProject();
+            Path hubProjDir = getDataHubAdminConfig().getHubProject().getProjectDir();
+            Assertions.assertTrue(hubProjDir.resolve("input").toFile().exists());
+            project.exportProject();
+            //copy file to build dir from project dir
+            FileUtils.copyFileToDirectory(hubProjDir.resolve("build").resolve("datahub-project.zip").toFile(), new File("build"));
```

Comment:
One benefit of exportProject requiring an input is that the client no longer needs to assume what the File is - the client will now determine that.